### PR TITLE
Homework 6

### PR DESCRIPTION
**Pull request by hirschvladimir**

### CODE REVIEW

File: homework_6/task01/oop_2.py [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+Всем перечисленным выше атрибутам и методам классов сохранить названия.
+К названием остальных переменных, классов и тд. подходить ответственно -
+давать логичные подходящие имена.
+"""
+from collections import defaultdict
+from datetime import datetime, timedelta
+from typing import Optional, Union
+
+
+class FullNameCreate:
```
Comment (AlekseiAfanasev) [inline homework_6/task01/oop_2.py]: class name like `Person` or `Human` would fit more, what do you think?

File: homework_6/task01/oop_2.py [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+        return difference_time < self.deadline
+
+
+class HomeworkResult:
+    def __init__(self, homework: Homework, solution: str):
+        if not isinstance(homework, Homework):
+            raise TypeError("You gave a not Homework object.")
+        self.homework = homework
+        self.solution = solution
+        self.author = Student
```
Comment (AlekseiAfanasev) [inline homework_6/task01/oop_2.py]: are you sure about this line?

File: homework_6/task01/oop_2.py (hunk 1 above)
Comment (hirschvladimir) [reply homework_6/task01/oop_2.py]: Yes, name Person seems better

File: homework_6/task01/oop_2.py (hunk 2 above)
Comment (hirschvladimir) [reply homework_6/task01/oop_2.py]: In was written that "author - хранит объект Student".

Comment (AlekseiAfanasev) [reply homework_6/task01/oop_2.py]: Student is class here

**Review by AlekseiAfanasev — APPROVED** [review]

**Pull request merged**